### PR TITLE
Prettier homepage

### PR DESCRIPTION
**Pull request by zregvart**
Have a look at the commits, still a lot of work left to do, but perhaps a good start...

### CODE REVIEW

**Review by lcarva — APPROVED** [review]
It's beautiful! :rocket:

**Comment by simonbaird** [comment]
I think the centered body looks odd with the full width header. Looks better with this disabled:

```
div.body {
  margin-left: auto;
  margin-right: auto;
}
```

File: website/assets/css/custom.css [hunk 1]
```diff
@@ -65,6 +68,26 @@
...
+    .navbar-end > a.navbar-item:hover, .navbar-item.has-dropdown:hover .navbar-link {
+        color: white;
+    }
+}
+
+/* center the main content horizontally */
+.body {
+    margin-left: auto;
+    margin-right: auto;
+}
```
Comment (simonbaird) [inline website/assets/css/custom.css]: Oh yeah, this part.

**Comment by simonbaird** [comment]
Are you thinking we'll apply this look to the antora pages as well?

**Comment by simonbaird** [comment]
Looking good. FYI, I changed my mind and went back to a margin auto, see https://29a3617a.enterprise-contract.pages.dev/ .

**Comment by zregvart** [comment]
Most of this is merged via  #56

**Pull request closed without merging**